requirements/AUR_CBI_INT-0230/booleanparam.md

```diff
@@ -0,0 +1,19 @@
+# Testing function with boolean type param
```

Comment:
Given that there is no md5 file and other QGenC related producs, isn't the slx model regenerated?

Reply:
No because simulink model is present, see previous projects